dashboard/modules/tune/tune_head.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import datetime
+import copy
+import os
+import aiohttp.web
+
+import ray.new_dashboard.modules.tune.tune_consts \
+    as tune_consts
+import ray.new_dashboard.utils as dashboard_utils
+from ray.new_dashboard.utils import async_loop_forever, rest_response
+
+try:
+    from ray.tune import Analysis
+    from tensorboard import program
+except ImportError:
+    Analysis = None
+
+logger = logging.getLogger(__name__)
+routes = dashboard_utils.ClassMethodRouteTable
+
+
+class TuneController(dashboard_utils.DashboardHeadModule):
+    def __init__(self, dashboard_head):
+        """
+        This dashboard module is responsible for enabling the Tune tab of
+        the dashboard. To do so, it periodically scrapes Tune output logs,
+        transforms them, and serves them up over an API.
+
+        Args
+            logdir (str): Directory path to save the status information of
```

Comment:
The `Args` comments can be removed? Or, can we move the comments to `self._logdir`?

Reply:
Good point, thank you.